docs/modules/mapstore/pages/configuring-a-generic-maploader.adoc

```diff
@@ -4,13 +4,32 @@
 
 {description}
 
+NOTE: The objects created in the distributed map are stored in a GenericRecord. You can build a map interceptor to store the data in a POJO (Plain Old Java Object).
```

Comment:
No, using interceptor is an incorrect approach. You should use the `type-name` property to store objects as POJOs.

Also the objects in the maps are stored *as*  `GenericRecord` not *in* `GenericRecord`

Reply:
Ah, so is the advice in the SUP out of date then? Will update